package.json

```diff
@@ -8,8 +8,8 @@
   "private": true,
   "scripts": {
     "wp-env": "wp-env",
-    "setup:tools": "yarn wp-env run composer install",
-    "test": "yarn wp-env run phpunit phpunit -c /var/www/html/wp-content/plugins/wporg-two-factor/phpunit.xml.dist",
+    "setup:tools": "yarn wp-env run cli --env-cwd=wp-content/plugins/wporg-two-factor composer install",
```

Comment:
I missed this and didn't run it, just `composer install && yarn test` worked for me

Reply:
I think this depends on the version you have installed locally; I found running composer via the containers was more reliable.

The composer is also expecting to be able to modify `../../../mu-plugins/pub` which won't be in the right place when run outside of the containers.